selfservice/strategy/oidc/provider_google.go

```diff
@@ -66,3 +70,27 @@ func (g *ProviderGoogle) AuthCodeURLOptions(r ider) []oauth2.AuthCodeOption {
 
 	return options
 }
+
+var _ IDTokenVerifier = new(ProviderGoogle)
+
+func (p *ProviderGoogle) Verify(ctx context.Context, rawIDToken string) (*Claims, error) {
+	keySet := oidc.NewRemoteKeySet(ctx, p.JWKSUrl)
+	verifier := oidc.NewVerifier("https://accounts.google.com", keySet, &oidc.Config{
+		ClientID: p.config.ClientID,
+	})
+	token, err := verifier.Verify(oidc.ClientContext(ctx, otelhttp.DefaultClient), rawIDToken)
```

Comment:
Please use the HTTP client form the registry, not the DefaultClient. Please fix this for apple too!

Reply:
Done.

Reply:
Evidently not ;)

Reply:
thanks... probably a bit flip ;)

Reply:
git flip? :D

Reply:
it's still not changed?

Reply:
now it is